apps/src/templates/sectionAssessments/assessmentsTestHelpers.js

```diff
@@ -80,3 +80,56 @@ export const multipleChoiceData = [
   },
 ];
 
+export const surveyOne = [
+  {
+    id: 1,
+    studentId: '210',
```

Comment:
Why are studentIds strings? I would've thought they'd be numbers?

Reply:
No specific reason.  The studentId is not currently utilized in this table, so I made it a string. Thoughts!  The design of the database will influence if the studentId should be number.